### DISCUSSION TITLE
Sales Order Allocating multiple serial tracked parts

### DISCUSSION THREAD

**Discussion by codeghari** [inventree/InvenTree#5835] [Q&A] [answered]
Hi everyone,

I have parts that we track via MAC addresses. 

I create a sales order, pick a part and then try to allocate stock to this sales order part. 

In the allocate stock items to sales order menu, I am presented with the part, a drop down menu to select one of the serial numbers and also a quantity option.

The problem I have is I cannot select multiple serial numbers, only 1 at a time. This makes allocating a lot of parts an impossible task if they are serially tracked. I feel like I am simply doing something wrong but it doesn't allow me to select more than one serial number at a time. 

Perhaps another way I thought of doing it was to scan objects into the search bar in the parts section. I figured if they were comma separated, the search would present each option and then I could just move the parts to the customers account. Not quite a sales order but we can work with it. This does not work the way I thought it would either. I can't seem to really use a barcode scanner to select multiple serial numbers/parts in any reasonable way at this stage.

**Comment by SchrodingersGat** [comment, marked as answer]
@codeghari you can allocate via multiple serial numbers using the "allocate serial numbers" button:

![image](https://github.com/inventree/InvenTree/assets/10080325/6525e913-0506-473d-97ef-1b63deea5bb4)

![image](https://github.com/inventree/InvenTree/assets/10080325/57812ae7-d57d-4815-838a-bf129968a761)


![image](https://github.com/inventree/InvenTree/assets/10080325/5da943ac-05d1-40a2-8212-badc3289d13e)

![image](https://github.com/inventree/InvenTree/assets/10080325/c9b95226-c602-4ad9-8bf3-9d7cc0a759fd)

**Reply by codeghari** [reply]
That's perfect. Thank you so much for the clear outline. I am sorry it was such a simple lack of knowledge!
